frontend/src/components/search/SearchFundingOpportunity.tsx

```diff
@@ -0,0 +1,81 @@
+import { Accordion, Checkbox } from "@trussworks/react-uswds";
+import React from "react";
+
+interface AccordionItemProps {
+  title: React.ReactNode | string;
+  content: React.ReactNode;
+  expanded: boolean;
+  id: string;
+  headingLevel: "h1" | "h2" | "h3" | "h4" | "h5" | "h6";
+  className?: string;
+  // handleToggle?: (event: React.MouseEvent<HTMLButtonElement>) => void;
+}
+
+interface FilterOption {
+  id: string;
+  label: string;
+  value: string;
+}
+
+export default function SearchFundingOpportunity() {
+  const filterOptions: FilterOption[] = [
+    {
+      id: "funding-opportunity-cooperative_agreement",
+      label: "Cooperative Agreement",
+      value: "cooperative_agreement",
+    },
+    { id: "funding-opportunity-grant", label: "Grant", value: "grant" },
+    {
+      id: "funding-opportunity-procurement_contract",
+      label: "Procurement Contract ",
+      value: "procurement_contract",
+    },
+    { id: "funding-opportunity-other", label: "Other", value: "other" },
+  ];
+
+  const accordionOptions: AccordionItemProps[] = [
+    {
+      title: "Funding instrument type",
+      content: (
+        <>
+          <div className="grid-row">
+            <div className="grid-col-fill">
+              <button className="usa-button usa-button--unstyled font-sans-xs">
+                Select All
+              </button>
+            </div>
+            <div className="grid-col-fill text-right">
+              <button className="usa-button usa-button--unstyled font-sans-xs">
+                Clear All
+              </button>
```

Comment:
Were you planning to handle the select and clear ina separate pr?

Reply:
Yeah. I wasn't handling any functionality here. Just getting the scaffolding in there.